features/bootstrap/LanguageSelectionContext.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Centreon\Test\Behat\Administration\ParametersCentreonUiPage;
+use Centreon\Test\Behat\Administration\ParametersMyAccountPage;
+use Centreon\Test\Behat\Configuration\CurrentUserConfigurationPage;
+use Centreon\Test\Behat\CentreonContext;
+
+class LanguageSelectionContext extends CentreonContext
+{
+    private $currentPage;
+
+    /**
+     * @Given I go to my account page
+     */
+    public function theUserWithAutologinEnabled()
+    {
+        $this->currentPage = new ParametersMyAccountPage($this);
+        $this->currentPage->save();
+    }
+
+    /**
+     * @When I select the language dropdown
+     */
+    public function selectTheLanguageDropdown()
+    {
+        $this->visit('main.php?p=50104&o=c');
```

Comment:
you could instantiate ParametersMyAccountPage

Reply:
changed as requested